src/UI/GameOptionSetupController.java

```diff
@@ -19,7 +19,7 @@ public class GameOptionSetupController implements IController
 
   private GameOption<FogMode> fowOption = new GameOption<FogMode>("Fog of War", FogMode.values(), 0);
   private GameOption<Integer> startingFundsOption = new GameOptionInt("Starting Funds", 0, 50000, 1000, 0);
-  private GameOption<Integer> incomeOption = new GameOptionInt("Income", 250, 20000, 250, 1000);
+  private GameOption<Integer> incomeOption = new GameOptionInt("Income", 0, 20000, 250, 1000);
```

Comment:
RIP Colin, I guess

Reply:
Or maybe he'd do better since could make more money with his power? Hm. Maybe RIP Kanbei.

Reply:
No, Colin can't create money if he doesn't have money.
Kanbei, OTOH, just wins with the predeployed units.